neo/Wallets/Wallet.py

```diff
@@ -894,16 +894,21 @@ def GetDefaultContract(self):
         Get the default contract.
 
         Returns:
-            contract (Contract): if Successful, a contract of type neo.SmartContract.Contract, otherwise None
+            contract (Contract): if Successful, a contract of type neo.SmartContract.Contract, otherwise an Exception.
+
+        Raises:
+            Exception: if no default contract is found.
 
         Note:
             Prints a warning to the console if the default contract could not be found.
         """
         try:
-            return self.GetContracts()[0]
-        except Exception as e:
-            logger.error("Could not find default contract")
-        return None
+            default = self.GetContracts()[0]
+        except Exception:
+            error_msg = "Could not find default contract"
+            logger.error(error_msg)
```

Comment:
Please include the original exception, and re-raise the original exception. Like this:

    except Exception as e:
        logger.error("Could not find default contract")
        logger.exception(e)
        raise